packages/react-components/react-combobox/src/components/Combobox/useComboboxStyles.styles.ts

```diff
@@ -212,6 +213,9 @@ const useIconStyles = makeStyles({
       display: 'block',
     },
   },
+  hidden: {
+    display: 'none',
```

Comment:
I think my last comment may have been missed, but this style can't use `display: none` or `visibility: hidden`, since those would also remove it from the accessibility tree. We have a couple examples of accessibly hidden CSS styles in [useAriaLiveStyles](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-toast/src/components/AriaLive/useAriaLiveStyles.styles.ts#L14-L20) or [useCardStyles](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-card/src/components/Card/useCardStyles.styles.ts#L347-L353)

Reply:
@smhigley there are two icons in `Combobox` case:
- `expandIcon` (has `role` and _visible_ for keyboard nav)
- `clearIcon` (has `aria-hidden`)

I added `.visuallyHidden` styles and use it only for `expandIcon`. `clearIcon` still gets `display: none`. 

Does it make sense?

Reply:
yup, perfect!